packages/terra-form/docs/Field.md

```diff
@@ -10,6 +10,7 @@ import Field from 'terra-form/lib/Fieldset';
 
 <Field
   label="Do you have any Children?"
+  labelAttrs={{ className: 'healtheintent-application' }}
   error="This field is required"
   help="Families are eligible for family package plans"
   required
```

Comment:
If we go with styled components, I would see the 3rd option as the result api most likely. From looking at styled components, I don't think we would have the need potentially to attach individual classes to the elements to get the desired styles, as they could just style the Label they provided to Label prop, and use those styles as is.

As for option 2, the main thing I'm concerned for on that one is that Fields are designed to be an easy API where developers don't need to worry about having to determine where the label and error messages go, and the second option brings that concern back. We do have those components in place that allow for consumers to build their own Fields if they need the label, help, and error in different places, so I lean more towards having consumers build that if they need it.